### ISSUE TITLE
[Bug] Only two sources for lyrics are working

### ISSUE THREAD

**Issue by serjflint** [clementine-player/Clementine#6677]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Windows 10
- Clementine version: 1.4.0rc1-206-g8f56fbb83

### Expected behaviour / actual behaviour

Lyrics from several sources / Lyrics only from lyrics.wikia.com, lyrics.com and tags

### Steps to reproduce the problem (only for bugs)

Enable different sources:
![2020-05-20_20-56-48](https://user-images.githubusercontent.com/4900246/82468659-84d11900-9adc-11ea-87c0-b977b619e846.jpg)
Got only two working:
![2020-05-20_20-57-10](https://user-images.githubusercontent.com/4900246/82468789-a7fbc880-9adc-11ea-8180-0c4582c61f02.jpg)
I can show tens of popular songs with such behaviour. When I disable lyrics.wikia.com and lyrics.com there are no lyrics at all except for tags.

**Comment by jbroadus** [comment]
Seems that several of these aren't around anymore. And at least one (azlyrics) is using recaptcha to prevent scraping.

**Comment by serjflint** [comment]
Then we should remove it or add some clutch. What is the point of having tens of dead providers?

**Comment by jbroadus** [comment]
Yep. Might be better to host the master list on clementine-player.org. At the very least, we should add a test that occasionally checks all of hard-coded URLs in the codebase.

**Comment by mepreston** [comment]
I don't see any lyrics sources working.  if this isn't resolved soon i will be swtich to another app

**Comment by dougtheold** [comment]
As 'mepreston' said (and might be the desired outcome), Strawberry is KDE's clone of Clememtine and is available in Discover (KDE), where Clementine is not.  KDE is adopting Microsoft business practices, maybe.
